athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneMetadataHandler.java

```diff
@@ -80,6 +94,10 @@ public class NeptuneMetadataHandler extends GlueMetadataHandler
     private final AWSGlue glue;
     private final String glueDBName;
 
+    private NeptuneConnection neptuneConnection = null;
+    SchemaBuilder schemaBuilder = null;
```

Comment:
isn't schema going to change per table request? why are we declaring it here?

Reply:
Hi @chngpe Thnak you, Removed from here and modified buildSchema function as per your suggestion.